jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/JfrMetrics.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.jfr.metrics;
+
+import io.opentelemetry.api.metrics.MeterProvider;
+import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jdk.jfr.EventSettings;
+import jdk.jfr.consumer.RecordingStream;
+
+public final class JfrMetrics {
```

Comment:
Would be good if you can add at least some simple javadoc to the class and enable method